src/Layout/QueryParamContext.tsx

```diff
@@ -274,6 +280,8 @@ export default function QueryParamProvider({ children }: any) {
         }
 
         setpaymentSetup(paymentSetupData.data);
+        setProjectName(paymentSetupData.data.name);
```

Comment:
lines 264-279

this is refetching the payment Options with project guid.

this is redundant, we do not need to wait for projectGUID to fetch the payment options
it can be fetched with `to={projectSlug}`

https://app.plant-for-the-planet.org/app/projects/${slug}/paymentOptions?country=DE